sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -37,7 +37,6 @@ Shared | No | Yes | Yes (default always)| No | No | No
 AAD | Yes | Yes | Yes | Yes | Optional | No
 Isolated | Yes| Yes | Yes | Yes | Yes | Yes
 
-
```

Comment:
Commenting here, since it's not part of the diff: on line 15, should this be "The Async versions of the samples require Python 3.6 or later." since this package doesn't support 3.5?

Reply:
Also, on line 19, `[sample_authentication_async.py][sample_authentication_async.py]` should be `[sample_authentication_async.py][sample_authentication_async]`

Reply:
Actually the correct version was the one without the .py, but the links below were incorrect. It all seems to work ok now.